app/controllers/articles_controller.rb

```diff
@@ -22,7 +23,8 @@ def create
       article=Article.new(
       title: article_params[:title],
       body: article_params[:body],
-      author: article_params[:author]
+      author_id: article_params[:author_id],
```

Comment:
instead of getting author_id from params, try to find author using email and then fill author id

Reply:
Changes implemented, please check latest commit.